helm/api-testing/values.yaml

```diff
@@ -5,10 +5,16 @@
 replicaCount: 1
 
 image:
+  registry: docker.io
   repository: linuxsuren/api-testing
-  pullPolicy: IfNotPresent
+  tag: "v0.0.2-helm"
```

Comment:
`v0.0.2-helm` is the tag of the helm image instead of the application. I perfer to use the chart appVersion as the default version instead of a fixed version.

Reply:
The fixed version is indeed not dynamic enough, but it is not user-friendly because after I download the chart, I still need to look at the code to know which image tag I am using.

Reply:
But the tag `v0.0.2-helm` is incorrect.